packages/proxy/src/providers/openai.ts

```diff
@@ -0,0 +1,249 @@
+// This is copied from the Vercel AI SDK commit bfa1182c7f5379d7a3d81878ea00ec84682cb046
+// We just need the OpenAI parser, but not the streaming code.
+
+import { CompletionUsage, FunctionCall, trimStartOfStreamHelper } from "ai";
+
+// https://github.com/openai/openai-node/blob/07b3504e1c40fd929f4aae1651b83afc19e3baf8/src/resources/chat/completions.ts#L28-L40
+interface ChatCompletionChunk {
+  id: string;
+  choices: Array<ChatCompletionChunkChoice>;
+  created: number;
+  model: string;
+  object: string;
+}
+
+// https://github.com/openai/openai-node/blob/07b3504e1c40fd929f4aae1651b83afc19e3baf8/src/resources/chat/completions.ts#L43-L49
+// Updated for https://github.com/openai/openai-node/commit/f10c757d831d90407ba47b4659d9cd34b1a35b1d
```

Comment:
Is this comment superseded by the next line?

Reply:
Not sure. It's just copy/pasted from Vercel's code. Would rather not change it.